x-pack/test/tsconfig.json

```diff
@@ -108,5 +108,6 @@
     "@kbn/core-provider-plugin",
     "@kbn/core-saved-objects-common",
     "@kbn/alerting-fixture-plugin",
+    "@kbn/user-profile-components",
```

Comment:
is this relevant to this PR?

Reply:
Good catch, @dgieselaar. It's not. Deleting